sakai-evaluation-tool/src/webapp/content/templates/modify_hierarchy_node_groups.html

```diff
@@ -67,13 +76,16 @@ <h3 rsf:id="assign-groups-title">Assign groups for a hierarchy node</h3>
                     <td>
                         <label rsf:id="group-title">Group Title</label>
                     </td>
+                    <td><label rsf:id="sections">List of sections goes here...</label></td>
                 </tr>
             </tbody>
         </table>
-        <p class="act"> 
-            <input type="submit" value="Save Assigned Groups" rsf:id="save-groups-button" class="active" accesskey="s" />
-            <a href="" rsf:id="cancel-link">Cancel</a> 
-        </p>    
+        <div class="action">
+            <span class="cancelButton">
+                <a href="#" rsf:id="return-link2">Return to Control Hierarchy</a>
+                <input type="submit" value="Save Assigned Groups" rsf:id="save-groups-button2" class="active" accesskey="s" style="float: right;" />
```

Comment:
Same as above comment; should likely be addressed in a separate JIRA project wide.